resources/js/pages/game/[game]/comments.tsx

```diff
@@ -10,7 +10,7 @@ const GameComments: AppPage<App.Community.Data.GameCommentsPageProps> = ({ game
 
   return (
     <>
-      <Head title={t('Comments: :game', { game: game.title })}>
+      <Head title={t('Comments: :gameTitle', { gameTitle: game.title })}>
```

Comment:
Suggest `Comments - :gameTitle` for consistency with Supported Game Files, Manage Hashes, and Manage Claims pages.

Reply:
No problem, updated in latest.